src/main/scala/com/womenwhocode/web/Webapp.scala

```diff
@@ -43,25 +52,20 @@ object Webapp {
       Ok(s"locations endpoint")
 
     case GET -> Root / "locations" / postcode =>
-      val getRequestTask = httpClient.expect[PostcodeResponse](s"http://api.postcodes.io/postcodes/${postcode}")
-      getRequestTask.flatMap(json => Ok(json))
+      Ok(get[PostcodeResponse](postcode))
 
     case GET -> Root / "locations" / postcode / "nearest" =>
-      val getRequestTask = httpClient.expect[MultiPostcodeResponse](s"http://api.postcodes.io/postcodes/${postcode}/nearest")
-      getRequestTask.flatMap(response => Ok(responseJson(response)))
+      for {
+        nearest <- get[MultiPostcodeResponse](s"$postcode/nearest")
+        resp <- Ok(responseJson(nearest))
+      } yield resp
```

Comment:
I think this is a good example.
In the tutorial, I will need to break up this up into steps and then introduce a refactor